### PR TITLE
[DiskRP] Adding swagger spec for 09-30-2018

### PR DESCRIPTION
**Pull request by realschwa**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
@pollock4595 I don't see this API version being added to the readme.md file, is that intentional?

**Comment by pollock4595** [comment]
> @pollock4595 I don't see this API version being added to the readme.md file, is that intentional?

@jhendrixMSFT Sorry I had forgotten  to commit that specific file. Adding it now.

**Comment by jhendrixMSFT** [comment]
@sarangan12 can you please take a look at the linter failure, I believe it's due to the hard-coded version of AutoRest and there's a newer version available so we get the "there's a new version of AutoRest..." spew.

**Comment by jhendrixMSFT** [comment]
@pollock4595 I checked in a fix for the linter issue, can you please rebase on top of master?

**Comment by ravbhatnagar** [comment]
Signing off from ARM side.

**Comment by jhendrixMSFT** [comment]
@pollock4595 the latest linter failure has also been fixed in master, could you please rebase again?  Sorry for the hassle.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2018-09-30/examples/CreateACustomImageScaleSetFromAnUnmanagedGeneralizedOsImage.json [hunk 1]
```diff
@@ -0,0 +1,202 @@
+{
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2018-09-30/examples/CreateACustomImageScaleSetFromAnUnmanagedGeneralizedOsImage.json]: Under 2018-09-30 is there only disk.json.  Remove all example files that are not referred by disk.json.

**Comment by pollock4595** [comment]
@jhendrixMSFT could the linter issue be a problem on my end since it keeps popping up?

There error is about a file I did not modify though.

File: specification/compute/resource-manager/readme.md [hunk 2]
```diff
@@ -177,15 +177,21 @@ directive:
     suppress:
       - DefinitionsPropertiesNamesCamelCase
 
-  - where:
```
Comment (jhendrixMSFT) [inline specification/compute/resource-manager/readme.md]: This removal is triggering the following ARM error, was it removed by mistake?
```
Potential new ARM errors
========================
R3026 - TrackedResourcePatchOperation
Tracked resource 'ContainerService' must have patch operation that at least supports the update of tags. It's strongly recommended that the PATCH operation supports update of all mutable properties as well.
  at specification/compute/resource-manager/Microsoft.ContainerService/stable/2017-01-31/containerService.json:555
```

File: specification/compute/resource-manager/Microsoft.Compute/stable/2018-09-30/disk.json [hunk 3]
```diff
@@ -0,0 +1,1311 @@
...
+          },
+          "description": "A collection of encryption settings, one for each disk volume."
+        }
+      },
+      "required": [
+        "enabled"
+      ],
+      "description": "Encryption settings for disk or snapshot"
+    },
+    "EncryptionSettingsElement": {
```
Comment (jhendrixMSFT) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2018-09-30/disk.json]: Can you add a description for this?  It's triggering the following linter warning.
```
Potential new SDK warnings
==========================
R4000 - DescriptionAndTitleMissing
'EncryptionSettingsElement' model/property lacks 'description' and 'title' property. Consider adding a 'description'/'title' element. Accurate description/title is essential for maintaining reference documentation.
  at specification/compute/resource-manager/Microsoft.Compute/stable/2018-09-30/disk.json:999
```

**Comment by jhendrixMSFT** [comment]
@pollock4595 is the new functionality live?

**Comment by pollock4595** [comment]
@jhendrixMSFT yep!

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**